example_projects/transfer/nlb_observability_stack/nlb.tf

```diff
@@ -0,0 +1,19 @@
+module "nlb_module" {
```

Comment:
I recommend to use datasource instead of this module

Reply:
removed module